src/pybind/mgr/dashboard/frontend/e2e/pools/pools.po.ts

```diff
@@ -85,15 +85,14 @@ export class PoolPageHelper extends PageHelper {
 
   @PageHelper.restrictTo(pages.index)
   async delete(name: string): Promise<any> {
-    const tableCell = await this.getTableCellByContent(name);
-    await tableCell.click();
+    await this.getTableCell(name).click();
     await $('.table-actions button.dropdown-toggle').click(); // open submenu
     await $('li.delete a').click(); // click on "delete" menu item
-    const confirmationInput = () => $('#confirmation');
-    await this.waitPresence(confirmationInput());
-    await this.clickCheckbox(confirmationInput());
```

Comment:
Do you have a particular reason for not using `clickCheckbox`? I think your versions works, too, but it might make sense to use `clickCheckbox` instead. It explains how to click checkboxes, ensures that the right type of element is clicked and complains properly if something goes wrong.

Reply:
It simply seems not to work. All other e2e tests that are deleting something do not use ``clickCheckbox``, and they work flawlessly.

Reply:
I'm confused about that. Surely, the method should have been used more often, but it's also used in the RBD image deletion. And the code worked several times. Do you think this is the critical part of your adaption to make this work? Because I'd really like to know which changes will fix the problem.

Reply:
To reduce the repetition of the waits, maybe we should put them inside the `clickCheckbox`, that way we always have the best approach to click checkboxes.
I would recommend using `waitPresence` for the element passed as an argument and after calculating the correct element to click we should use `waitClickable` on that one.